### ISSUE TITLE
Can not update default locale translation

### ISSUE THREAD

**Issue by chuckyuee** [spree/spree_backend#318]
## Context

![截屏2024-05-29 21 21 16](https://github.com/spree/spree_backend/assets/161027936/e26c5bb9-4457-4ef1-b137-6b2ba9c54e84)

Can not update site name to Spree Demo Store, but zh-CN translation is ok.

## Expected Behavior

Can update site name to Spree Demo Store.

## Actual Behavior

Nothing change

## Possible Fix

in Spree::Admin::Translatable class, save_translation_values method maybe ignore default locale translation, or direct to update default locale in main table not in translation table.

## Steps to Reproduce

1. login in admin
2. click settings > store
3. click translations
4. update form

## Your Environment

* Version used: 4.8.2

**Comment by chuckyuee** [comment]
```
      def save_translation_values
        translation_params = params[:translation].permit!.to_h

        current_store_locales = current_store.supported_locales_list
        params_locales = translation_params.flat_map { |_attribute, translations| translations.compact_blank.keys }
        locales_to_update = current_store_locales & params_locales

        locales_to_update.each do |locale|
          if !Spree.always_use_translations? && locale == I18n.default_locale.to_s
            translation_params.each do |attribute, translations|
              @object.public_send("#{attribute}=", translations[locale])
            end
            @object.save!
          else
            translation = @object.translations.find_or_initialize_by(locale: locale)
            translation_params.each do |attribute, translations|
              translation.public_send("#{attribute}=", translations[locale])
            end
            translation.save!
          end
        end
      end
```

**Comment by damianlegawiec** [comment]
@chuckyuee thanks for reporting and posting code - this is now released as `spree_backend` `v4.8.1`
